### DISCUSSION TITLE
Will the quick list be removed in the future?

### DISCUSSION THREAD

**Discussion by JobcenterTycoon** [uBlockOrigin/uBlock-issues#3018] [Q&A] [answered]
uBlock Origin supporting Differential updates since 1.54.0 so every uBO filterlist getting updated every 4 hours.

This will make the quick list obsolete. Or not?

**Comment by peace2000** [comment]
If something big breaks, one hour vs. 4 hours is a noticeable difference.

**Reply by JobcenterTycoon** [reply]
Hmm the template says `! Diff-Expires: 317 minutes` like evey other list.

**Reply by gorhill** [reply]
317 minutes is between 5 and 6 hours.

We shouldn't remove quick-updates since it still might need to update without diff-update after a manual update.

---

Well at least for the time being. Maybe in two months will see where server usage is at, then according to result maybe we could lower `! Expires` to 12 hours for all default lists.

**Reply by gorhill** [reply, marked as answer]
> Maybe in two months will see where server usage is at,

Thinking more about this, we can't remove the quick-fixes list because whenever high traffic sites require an update, it will always be better to update a small list to get the fix than the full filters.txt one, so we need to keep it forever really.

**Comment by BlazeFTL** [comment]
I Think Quick Fix Should Be Returned To Its Previous State Meaning No Diff Updates For It...So Its Still A "Quick Fix"

**Reply by u-RraaLL** [reply]
How does changing from 5 hours to 12 hours make it quicker?

For instant fixes, volunteers will be giving out update links for the users to click.

**Reply by BlazeFTL** [reply]
I Didnt Mean That...I Meant Me Clicking On Quick Fix Update 10 Minutes  After A Commit
